RELEASE.md

```diff
@@ -1,3 +1,83 @@
+# Release 1.15.0
+
+## Major Features and Improvements
+
+## Breaking Changes
+
+## Bug Fixes and Other Changes
+
+* Promoting `unbatch` from experimental to core API.
+* Adds `tf.enable_control_flow_v2()` and `tf.disable_control_flow_v2()`.
```

Comment:
remove here, covered by the section proposed in major features.

Reply:
Done.